client-java/BUILD

```diff
@@ -31,6 +31,8 @@ java_library(
         "@graknlabs_graql//java:graql",
         "//server:server",
 
+         "//dependencies/maven/artifacts/grakn/benchmark:lib",
```

Comment:
This should be replaced with Bazel, @flyingsilverfin. Make sure to plan the migration to Bazel soon.